specification/Metadata/Semantics/README.md

```diff
@@ -0,0 +1,73 @@
+# Cesium Metadata Semantic Reference
+
+This document defines a general set of semantics for 3D Tiles and glTF. Tileset authors may define their own application- or domain-specific semantics separately.
```

Comment:
There's not enough context here.

Is this a spec?

Or a non-normative reference?

And "general set" just is not enough context to under the **why** and the **what** for this spec.

Reply:
And diagram please showing how the components of the ecosystem come together.

Even placeholder Miro board screenshot would be fine then a graphics artist could improve things later.

Reply:
And these semantics are truly just for 3D Tiles and glTF?  Not anything else?  CZML in the future, etc.?

Reply:
I think this document is basically a dictionary then other specs can reference the specific vocab here instead of copy and pasting it.  Please state this explicitly and more precisely than I just did. 😄